components/BottomMenu.js

```diff
@@ -29,13 +29,14 @@ function BottomMenu ({ navigation }, props) {
 
     const getDestination = async () => {
         let searchItem = await AsyncStorage.getItem("toLocation");
-        if (searchItem.length > 13) {
-            var upadatedSearchItem = searchItem.substring(0, 13) + "...";
-            setDestination(upadatedSearchItem);
-        }
-        else {
-            setDestination(searchItem);
-        }
+        setDestination(searchItem);
+        // if(searchItem.length > 13){
```

Comment:
This removes the functionality of setting the destination from the search bar. I will create a PR for the fix, just merge your branch with master and push again when you approve the fix.

Reply:
it does not?